config/config-package.py

```diff
@@ -48,6 +49,20 @@ def call(*args, capture_output=False):
     with open(config_type_path / 'packages.txt', 'a') as f:
         f.write(f'{path.name}\n')
 
+
+# Read and update meta configuration
+meta_cfg = ConfigParser()
+meta_cfg_path = path / '.meta.cfg'
+if meta_cfg_path.exists():
+    meta_cfg.read(meta_cfg_path)
+else:
+    meta_cfg['meta'] = {}
```

Comment:
I think `fail-under` should be initialized to `0`. This makes the code using it easier.

Reply:
Done.